docs/pages/eas-metadata/getting-started.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Getting started with EAS Metadata
+sidebar_title: Getting started
+---
+
+import { Terminal } from '~/ui/components/Snippet';
+
+> ⚠️ EAS Metadata is in beta and subject to breaking changes.
+
+EAS Metadata helps you prepare your app for review by uploading most of the required app information using a simple JSON file. It also helps you prevent common pitfalls that may lead to a rejected app submission.
+
+## Prerequisites
+
+EAS Metadata is available starting from EAS CLI >= 0.54.0, and _currently_ only supports the Apple App Store.
+
+If you are using vscode, also make sure to [install the Expo plugin](https://github.com/expo/vscode-expo) for **store.config.json** auto-completion.
+
+## Create the store config
+
+Let's start by creating our **store.config.json** file in the root directory of your project. This file holds all the information you want to upload to the app stores. If you have an existing app in the stores, you can generate the config file by running:
+
+<Terminal cmd={['$ eas metadata:pull']} />
+
+If you don't have an app in the stores, EAS Metadata can't pull that information. You can create a new **store.config.json** file manually instead.
+
+```json
+{
+  "configVersion": 0,
+  "apple": {
+    "info": {
+      "en-US": {
+        "title": "Awesome App",
+        "subtitle": "Your self-made awesome app",
+        "description": "The most awesome app you've ever seen",
+        "keywords": ["awesome", "app"],
+        "marketingUrl": "https://example.com/en/promo",
+        "supportUrl": "https://example.com/en/support",
+        "privacyPolicyUrl": "https://example.com/en/privacy"
+      }
+    }
+  }
+}
+```
+
+> EAS Metadata will use the **store.config.json** file name in your project root by default; you can change the name and location of the file by [configuring the `metadataPath` on the EAS Submit profile](../submit/eas-json.md#metadatapath).
+
+## Update the store config
+
+Now it's time to edit the **store.config.json** file and customize it to your app needs. A complete overview of what can be defined is coming soon. In the meantime, you can use the [vscode Expo plugin](https://github.com/expo/vscode-expo) and use the auto-completion.
+<!-- Disabled due to missing config overview
+You can find all available options for the **store.config.json** in the [store configuration reference](./store-json.md). -->
+
+## Upload a new app version
+
+Before we can upload the **store.config.json**, you must upload a new binary of your app. You can [read more about uploading new binaries to stores here.](../distribution/uploading-apps.md).
```

Comment:
```suggestion
Before we can upload the **store.config.json**, you must upload a new binary of your app. You can [read more about uploading new binaries to stores here.](../distribution/uploading-apps.md).
```
```suggestion
Before you can upload the **store.config.json**, you must upload a new binary of your app. [Read more about uploading new binaries to stores](../distribution/uploading-apps.md).
```

As per our Docs Writing Style, let's stick to [writing in second person](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md#write-in-the-second-person).